pasha_modules/util.coffee

```diff
@@ -6,32 +6,23 @@ constant = require('../pasha_modules/constant').constant
 State = require('../pasha_modules/model').State
 nodemailer = require "nodemailer"
 moment = require('moment')
+request = require('request')
 
 ack = ['roger', 'roger that', 'affirmative', 'ack', 'consider it done', 'done', 'aye captain']
 
 downloadUsers = (token, setUsersCallback)->
     scribeLog "downloading users"
-    try
-        options = {
-            hostname: "api.hipchat.com"
-            port: 443
-            path: "/v1/users/list?format=json&auth_token=#{token}"
-            method: "GET"
-        }
-        https.get options, (res) ->
-            data = ''
-            res.on 'data', (chunk) ->
-                data += chunk.toString()
-            res.on 'end', () ->
-                users = JSON.parse(data)["users"]
-                setUsersCallback(users)
-                scribeLog "downloaded #{users.length} users"
-    catch error
-        scribeLog "ERROR downloadUsers #{error}"
-        setUsersCallback([])
+    slackApi("users.list", {token: constant.slackApiToken}, (err, res, data) ->
+        if err || !data.ok
+          scribeLog "ERROR downloadUsers #{err || data.error}"
+          setUsersCallback([])
+        else
+          setUsersCallback(data.members)
+          scribeLog "downloaded #{data.members.length} users"
+    )
```

Comment:
Slight difference in our style (feel free to ignore this comment): I'd rather remove the outer parentheses.